### ISSUE TITLE
restore the command line client

### ISSUE THREAD

**Issue by kostja** [tarantool/tarantool#1288]
Implement a quick way to connect to a tarantool instance from the command line.
The entry point to the function is ```tarantoolctl connect```.

**Comment by rtsisyk** [comment]
you has convinced  me that `tarantoolctl connect` is not so big deal.
OK for 1.6.9.

**Comment by bigbes** [comment]
> you has convinced me that `tarantoolctl connect` is not so big deal.

It is a big deal, it must be `tarantoolcli`, `ctl` - means control, `cli` -
means command line interface.

With best regards, Eugene.

2016-04-01 17:53 GMT+03:00 Roman Tsisyk <notifications@github.com>:

**Comment by racktear** [comment]
See 9db1f9896992f0d5c323d181cb24849b55c2d4dc

**Comment by rtsisyk** [comment]
Implemented:

```
[roman@work 1.6]$ tarantoolctl connect 'unix/:/var/run/tarantool/example.control'
/usr/bin/tarantoolctl: connected to unix/:/var/run/tarantool/example.control
unix/:/var/run/tarantool/example.control> 
---
...

unix/:/var/run/tarantool/example.control> 
[roman@work 1.6]$ tarantoolctl connect '127.0.0.1:3313'
/usr/bin/tarantoolctl: connected to 127.0.0.1:3313
127.0.0.1:3313> 

```

Please note that unix pipes are still not implemented properly (see #672):

```
[roman@work 1.6]$ echo "2 + 2" | tarantoolctl connect '127.0.0.1:3313'
/usr/bin/tarantoolctl: connected to 127.0.0.1:3313
127.0.0.1:3313> 2 + 2  <-- should not display prompt
---
- 4
...

127.0.0.1:3313> <-- should not display prompt
```
